config.go

```diff
@@ -856,7 +857,7 @@ func (c *Config) ValidateConfig() error {
 	}
 
 	if c.DBReadRetries == 0 {
-		c.DBReadRetries = 5
+		c.DBReadRetries = 60
```

Comment:
Implies `60 * 1s` retry schedule. I thought it's a reasonable starting point, similarly to how `gh-ost` does it ([60](https://github.com/github/gh-ost/blob/801ebabe343621fc06c92667fdb404f24bc605c0/go/base/context.go#L288) and [1s](https://github.com/github/gh-ost/blob/801ebabe343621fc06c92667fdb404f24bc605c0/go/logic/migrator.go#L140))